src/components/ProductCard.jsx

```diff
@@ -0,0 +1,38 @@
+import placeholder from "../assets/placeholder.png";
+import heart from "../assets/heart.png";
+import styles from "./ProductCard.module.css";
+
+const ProductCard = ({ product }) => {
+  // 가격 포맷팅 (1000 -> 1,000)
+  const formatPrice = (price) => {
```

Comment:
💊 제안
코드로 바로 파악할 수 있는 주석은 불필요하니 지우시는 것을 추천드려요!

```suggestion
  const formatPrice = (price) => {
```